### DISCUSSION TITLE
How to fix / catch `flutter_test` error.

### DISCUSSION THREAD

**Discussion by Rohithgilla12** [fzyzcjy/flutter_convenient_test#244] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19389850/183377158-53a9a74e-2ab4-41fe-8e75-5d541afae9eb.png)

As the above image show, the `flutter_test` errors are being caught as errors of the test!

Is there any way to do it?
Because it is resulting in tests being failed :(

**Comment by fzyzcjy** [comment, marked as answer]
There are two things:

1. Seems your flutter test is really throwing an error. If you do not deliberately create that error, then it *should* cause your test to fail, instead of pass.
2. Customizing logic in https://github.com/fzyzcjy/flutter_convenient_test/blob/c2eddc9d0ae45c30f8f1de4e21eee2c3f1bdde6f/packages/convenient_test_dev/lib/src/functions/command.dart#L97 may be helpful to suppress them.
